.github/workflows/build-citus-community-nightlies.yml

```diff
@@ -23,16 +23,13 @@ jobs:
       fail-fast: false
       matrix:
         platform:
-          - el/7
           - el/8
-          - ol/7
           - debian/buster
           - debian/bullseye
           - debian/bookworm
           - ubuntu/bionic
           - ubuntu/focal
           - ubuntu/jammy
-          - ubuntu/kinetic
```

Comment:
@gurkanindibay could you double check these changes?

Reply:
We can remove ubuntu/kinetic however, we can not remove other ones. We still support el/7 and ol/7